doc/nrf/ug_thread_architectures.rst

```diff
@@ -67,35 +72,26 @@ It also has the following disadvantages:
 
 This platform design is suitable for the following development kits:
 
-+--------------------------------+-----------+------------------------------------------------+-------------------------------+
-|Hardware platforms              |PCA        |Board name                                      |Build target                   |
-+================================+===========+================================================+===============================+
-|:ref:`nRF52840 DK <ug_nrf52>`   |PCA10056   |:ref:`nrf52840dk_nrf52840 <nrf52840dk_nrf52840>`|``nrf52840dk_nrf52840``        |
-+--------------------------------+-----------+------------------------------------------------+-------------------------------+
-|:ref:`nRF52833 DK <ug_nrf52>`   |PCA10010   |:ref:`nrf52833dk_nrf52833 <nrf52833dk_nrf52833>`|``nrf52833dk_nrf52833``        |
-+--------------------------------+-----------+------------------------------------------------+-------------------------------+
-|:ref:`nRF5340 DK <ug_nrf5340>`  |PCA10095   |:ref:`nrf5340dk_nrf5340 <nrf5340dk_nrf5340>`    |``nrf5340dk_nrf5340_cpuapp``   |
-|                                |           |                                                |                               |
-|                                |           |                                                |``nrf5340dk_nrf5340_cpuappns`` |
-+--------------------------------+-----------+------------------------------------------------+-------------------------------+
-|nRF21540 DK                     |PCA10112   |:ref:`nrf21540dk_nrf52840 <nrf21540dk_nrf52840>`|``nrf21540dk_nrf52840``        |
-+--------------------------------+-----------+------------------------------------------------+-------------------------------+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf21540dk_nrf52840
 
 .. _thread_architectures_designs_soc_designs_multiprotocol:
 
 Single-chip, multiprotocol (SoC)
 ================================
 
-With nRF52 and nRF53 Series devices supporting multiple wireless technologies, including IEEE 802.15.4 and Bluetooth Low Energy (Bluetooth LE), the application layer and OpenThread still run on the same processor.
+nRF52 and nRF53 Series devices support multiple wireless technologies, including IEEE 802.15.4 and Bluetooth Low Energy (Bluetooth LE).
 
-In this multiprotocol design, the SoC ensures either dynamic or switched Thread and Bluetooth LE connectivity.
+In a single-chip, multiprotocol design, the application layer and OpenThread run on the same processor.
+The SoC ensures either dynamic or switched Thread and Bluetooth LE connectivity.
```

Comment:
It may be a good opportunity to drop this information.
In the old SDK we had two categories of samples - switched and dynamic.
In the switched variant one protocol (BLE/Thread) had to be completely disabled before the other one could be enabled. In dynamic both could operate concurrently.

As far as I know there are no switched samples present of planned for the NCS - this may be a leftover from the old documentation that we missed while moving it to the NCS.